src/Mod/Spreadsheet/App/PropertySheet.h

```diff
@@ -130,10 +130,14 @@ class SpreadsheetExport PropertySheet : public App::PropertyExpressionContainer
 
     void insertRows(int row, int count);
 
+    std::vector<App::CellAddress> getRows(int row, int count);
```

Comment:
Thanks for pointing out const, I'll update it accordingly :+1:

Regarding the name, I understand what you mean but I think these names are closer to the way other functions with similar return types are named in this class:
```c++
std::set<CellAddress> PropertySheet::getUsedCells() const
const std::set<CellAddress> &PropertySheet::getDeps(const std::string &name) const
```
Regarding adding a "-Range" suffix:
I've deliberately named them similar to `void PropertySheet::removeRows(int row, int count)` which takes the same input, so I don't think adding the suffix "-Range" is necessary.

I believe it might be better to follow the naming scheme in the file for this PR. Maybe it could use a refactoring as a later pass, but I don't think it should be done here.

Please let me know if you still think I should rename them. I'm open to doing it either way :)

Reply:
I approve of your reasoning. I don't demand you change anything (beyond `const`), but if you have spare time I'd approve of doing it 👍 
Realistically I don't think there'll be a separate refactor pass, things like that usually just don't happen :P

Reply:
I've pushed a new version which includes the `const` change as suggested.